CMakeLists.txt

```diff
@@ -370,18 +369,23 @@ endif()
 install(FILES CHANGELOG.md README.md
         DESTINATION ${CMAKE_INSTALL_PREFIX})
 
+set(MATERIALX_GEN_CONFIG_PATH "${CMAKE_INSTALL_PREFIX}/${MATERIALX_INSTALL_LIB_PATH}/cmake/${CMAKE_PROJECT_NAME}")
```

Comment:
Path as recommended in https://cmake.org/cmake/help/latest/guide/importing-exporting/index.html#exporting-targets